driver/gpio/imxgpio/imxgpio.cpp

```diff
@@ -1915,12 +1913,28 @@ NTSTATUS IMX_GPIO::configureCPUType (
         pinCount = IMX8M_GPIO_PIN_COUNT;
         pullShift = IMX8M_GPIO_PULL_SHIFT;
         pullMask = IMX8M_GPIO_PULL_MASK;
+        pullUp = IMX8M_GPIO_PULL_UP;
+        pullDown = (UINT32)-1;  // Force invalid value. iMX8M doesn't implement pull-down
```

Comment:
I think it'd be clearer to define an IMX_GPIO_PULL_INVALID in the IMX_GPIO_PULL enum, then explicitly list it above the default cases in PullModeToString and in GpioPullModeToImxPullMode.

Reply:
I've added a definition to the enumeration, and added it to PullModeToString, but it shouldn't be in GpioPullModeToImxPullMode.  IMX_GPIO_PULL reflects values we read or write to the IMX hardware + two flag values (DEFAULT and INVALID).  GpioPullModeToImxPullMode converts GPIO CLX value defs to IMX variants.